File: app/src/main/java/io/github/sithengineer/motoqueiro/util/ReleaseTree.java
package io.github.sithengineer.motoqueiro.util;

import android.util.Log;
import timber.log.Timber;

public class ReleaseTree extends Timber.DebugTree {

  @Override protected boolean isLoggable(String tag, int priority) {
    if (priority == Log.VERBOSE || priority == Log.DEBUG || priority == Log.INFO) {
      return false;
    }
    return true;
  }
}

File: app/src/main/java/io/github/sithengineer/motoqueiro/util/ReleaseTree.kt
package io.github.sithengineer.motoqueiro.util

import android.util.Log
import timber.log.Timber

class ReleaseTree : Timber.DebugTree() {

  override fun isLoggable(tag: String?, priority: Int): Boolean {
    return !(priority == Log.VERBOSE || priority == Log.DEBUG || priority == Log.INFO)
  }
}
